frigate/util.py

```diff
@@ -650,35 +650,45 @@ def restart_frigate():
 
 
 class EventsPerSecond:
-    def __init__(self, max_events=1000):
+    def __init__(self, max_events=1000, last_n_seconds=10):
         self._start = None
         self._max_events = max_events
+        self._last_n_seconds = last_n_seconds
         self._timestamps = []
 
     def start(self):
         self._start = datetime.datetime.now().timestamp()
 
     def update(self):
+        now = datetime.datetime.now().timestamp()
         if self._start is None:
-            self.start()
-        self._timestamps.append(datetime.datetime.now().timestamp())
+            self._start = now
```

Comment:
If this is going to be done this way then `start()` should be removed since it is not used

Reply:
start() is still called by video.py (frame_rate.start(), skipped_eps.start(), etc.)